assets/src/stories-editor/components/block-mover/ignore-nested-events.js

```diff
@@ -83,7 +83,7 @@ export class IgnoreNestedEvents extends Component {
 			// Try to match prop key as event handler
 			const match = key.match( /^on([A-Z][a-zA-Z]+?)(Handled)?$/ );
 			if ( match ) {
-				const isHandledProp = !! match[ 2 ];
+				const isHandledProp = Boolean( match[ 2 ] );
```

Comment:
^ Curious, which of the added rules enforces this?

Reply:
The [`no-implicit-coercion`](https://eslint.org/docs/rules/no-implicit-coercion) rule 🙂